Classes/Player/MediaEntry.swift

```diff
@@ -22,6 +22,7 @@ public class MediaEntry: NSObject {
     public var sources: [MediaSource]?
     public var duration: Int64?
     public var mediaType: MediaType?
+    public var metadata:[(name:String,value:String)]?
```

Comment:
As discussed, this should be a dictionary, defined as:
`public var metadata: [String:String]?`
Is the value guaranteed to be a String?